docs/tls/README.md

```diff
@@ -107,31 +162,43 @@ subjectAltName = @alt_names
 IP.1 = 127.0.0.1
 ```
 
-Then run
+#### 3.3.2 <a name="specifyconfigfile"></a>Run `openssl` and specify the configuration file:
```

Comment:
Change anchor to reflect title

Reply:
I've just removed the anchors from section 3.3, as they were not used.